### DISCUSSION TITLE
Web page: share → annotate + ref

### DISCUSSION THREAD

**Discussion by ildar** [gsantner/markor#1993] [Ideas] [answered]
I've seen this: https://www.youtube.com/watch?v=nH1ukQY3Ia8
This is exciting as it is quite desirable to have a new note with these:
1. Text excerpt from a web page
2. Annotation
3. tags
4. ref to the page

Yes, it's possible to do that all with two or three times of apps switching. But it better be quick and comfortable.

**Comment by gsantner** [comment, marked as answer]
Hello, 

see below for what amount of details i.e. browsers give when sharing i.e. a link from the browser to another app. That's all information available.


As what you suggested to is not exported by browsers, so this can't be added. And no, Markor won't get an addition to request website  on it's own to fetch more details. Markor doesn't make any unexpected network calls and it should stay like that.

![screenshot-2023-03-06__18-13-45](https://user-images.githubusercontent.com/6735650/223182496-fe47cc44-0200-4749-96e0-3276ced6fd12.png)
